content/en/docs/languages/java/automatic/spring-boot.md

```diff
@@ -165,8 +165,17 @@ IDE.
 
 The OpenTelemetry Starter supports all the
 [SDK Autoconfiguration](/docs/languages/java/automatic/configuration/#sdk-autoconfiguration)
-(since 2.2.0). You can set properties in the `application.properties` or the
-`application.yaml` file, or use environment variables.
+(since 2.2.0).
+
+{{% alert title="Attention" color="warning" %}}
+
+Unlike the SDK autoconfiguration, the OpenTelemetry Spring Boot starter uses
+`http/protobuf` as the default protocol, not `grpc`.
```

Comment:
Can we update the source where the defaults are listed instead? and to mention there the defaults for both the Java agent and Spring boot starter?

Reply:
done